Danki2/Assets/Scripts/Stats/StatsManager.cs

```diff
@@ -1,27 +1,43 @@
 ﻿using System;
+using System.Collections.Generic;
 
 public class StatsManager
 {
     private readonly Stats _baseStats;
-    private readonly Stats _frameStats;
+    private readonly Dictionary<Stat, float> _statsCache;
+    private List<Func<Stat, float, float>> _pipes;
 
     public StatsManager(Stats baseStats)
     {
         _baseStats = baseStats;
-        _frameStats = new Stats(_baseStats);
+        _statsCache = new Dictionary<Stat, float>();
```

Comment:
Small thing, but could just call this `_cache` as its already in the stats manager?

Reply:
I completely subscribe to this idea